packages/flat-components/src/components/CloudStorage/CloudStorageFileTitle/index.tsx

```diff
@@ -160,6 +185,9 @@ function getFileIcon(fileName: string): string {
         case ".flac": {
             return audioSVG;
         }
+        case "": {
+            return directorySVG;
+        }
```

Comment:
We are using a regex pattern to get the file extension here.

We could use [`path.extname()`](https://nodejs.org/dist/latest/docs/api/path.html#pathextnamepath) instead.

Reply:
We are Flat-web using the vite, unsupport the CJS.